test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -143,7 +143,6 @@ Describe "Verify approved aliases list" -Tags "CI" {
 "Alias",  "saps",                               "Start-Process",                      $($FullCLR -or $CoreWindows -or $CoreUnix),   "ReadOnly",         ""
 "Alias",  "sasv",                               "Start-Service",                      $($FullCLR -or $CoreWindows              ),   "ReadOnly",         ""
 "Alias",  "sbp",                                "Set-PSBreakpoint",                   $($FullCLR -or $CoreWindows -or $CoreUnix),   "ReadOnly",         ""
-"Alias",  "sc",                                 "Set-Content",                        $($FullCLR -or $CoreWindows -or $CoreUnix),   "ReadOnly",         ""
```

Comment:
Windows PowerShell only aliases are also included here so this table is a complete source of alias changes. So maybe this entry should be kept and only change the 4th column to be `$($FullCLR                               )` like `scb`.

Reply:
Yes, I'll make that change until we completely remove the Windows PowerShell code :)

Reply:
Fixed